suite-native/app/package.json

```diff
@@ -12,6 +12,8 @@
         "type-check": "yarn g:tsc -b",
         "pods": "npx pod-install",
         "prebuild": "expo prebuild",
+        "eas-build-post-install": "yarn build:libs && yarn root:messages",
```

Comment:
https://github.com/trezor/trezor-suite/pull/10898/commits/ffdda177dc654f11effb3b45beb83d4b6f476162

Reply:
`yarn messages` script run already as part of `yarn build:libs`, so it can be removed from here, see https://github.com/trezor/trezor-suite/blob/feat/eas-build/suite-common/message-system/package.json/#L9

Reply:
Yeah, good catch. Thanks
https://github.com/trezor/trezor-suite/pull/10898/commits/a6427c75993c7934ea3f70896d5254ed8ed37781